src/templates/tag/tagStructure.js

```diff
@@ -104,7 +105,7 @@ const fieldTypeDisplay = (field, fieldTypeStruct, metaIndex) => {
     typeCodeParts.push(` (${field.unit})`);
   }
   if (field.bounds) {
-    typeCodeParts.push(` (bounds)`);
+    typeCodeParts.push(` (min & max)`);
```

Comment:
Hopefully a bit more clear that the field is repeated (once for min, once for max).